src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,106 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import util.ConnectionUtill;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertQuery = "INSERT INTO manufacturers(name, country) values (?, ?);";
+        try (Connection connection = ConnectionUtill.getConnection();
+                PreparedStatement preparedStatement
+                        = connection.prepareStatement(insertQuery,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                Long id = resultSet.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't insert manufacturer " + manufacturer + " to DB", e);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String getQuery = "SELECT * FROM manufacturers where id = ? and NOT is_deleted;";
+        try (Connection connection = ConnectionUtill.getConnection();
+                    PreparedStatement preparedStatement = connection.prepareStatement(getQuery)) {
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            Manufacturer manufacturer = new Manufacturer();
```

Comment:
we should return empty Optional, not empty manufacturer in case there is no such manufacturer in db
```suggestion
            Manufacturer manufacturer = null;
```